src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs

```diff
@@ -42,7 +42,7 @@ public sealed class ScheduledJobDefinition : ISerializable, IDisposable
         private bool _isDisposed;
 
         // Task Action strings.
-        private const string TaskExecutionPath = @"powershell.exe";
+        private const string TaskExecutionPath = @"pwsh.exe";
```

Comment:
This code assumes a discoverable pwsh.exe native command.  Does this fix include adding pwsh.exe to the system path environment variable?  If so what version?  I wonder if we should allow the user to select a powershell.exe or pwsh.exe path to use for scheduled jobs (maybe in a separate issue/pr).

Reply:
Maybe we should use Utils.DefaultPowerShellAppBase here as we do with PowerShellProcessInstance.cs to ensure the scheduled job is using the correct path.
The problem with scheduled jobs is that they can run indefinitely on a machine and if pwsh.exe is uninstalled or installed to a new location then they will break.  Of course we have a similar problem with credentials expiring.

Reply:
I have a separate PR coming for the msi installer to append to %PATH% which would also solve the Win+R->pwsh usage

If PSCore6 is uninstalled, the full path won't help either.

Reply:
Actually, I'll do the %PATH% msi change as part of this PR as it's more useful that pwsh.exe differs form powershell.exe and I can put it at the end of the path rather than taking over the front